visualization/knit_table.R

```diff
@@ -27,7 +56,7 @@ library(scales)
 library(ggimage)
 library(cowplot)
 
-scIB_knit_table <- function(
+knit_table <- function(
```

Comment:
I don't feel it's necessary to use the original name given that it has been modified. I think that having `scIB` in the name helps make it clear that: 1) this is a modified version of an existing function for this project and 2) the function only works with scIB output data (there is a lot of hardcoded stuff). It's not a big deal though and @martaint can give her thoughts on how much it was modified.

Reply:
Ok! I changed back the function name and adapted the readme accordingly. I'll explain in the PR thread in more details.